server/sonar-server/src/main/java/org/sonar/server/component/ws/SearchAction.java

```diff
@@ -95,6 +101,13 @@ public void define(WebService.NewController context) {
       .setExampleValue("my-org")
       .setSince("6.3");
     action
+      .createParam(PARAM_ALL_ORGANIZATIONS)
+      .setDescription("Whether to return components of all organizations. Must not be set, if '" + PARAM_ORGANIZATION + "' is set.")
```

Comment:
My bad, validation is done by SearchWsRequest validate(). I didn't see because we try to not do any validation of request in WS client.